sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -244,6 +245,18 @@ abstract class RapidsMeta[INPUT <: BASE, BASE, OUTPUT <: BASE](
       }
     }
 
+    val isExp = wrapped.isInstanceOf[Expression]
```

Comment:
I think this makes more sense to be in `BaseExprMeta` where we know we have an expression.  IIUC all expressions should be using it.

Reply:
Also we have to allow Literal values to pass through this OK.  Literal values are foldable, but we support them

Reply:
isn't that covered by `exp.foldable && !GpuOverrides.isLit(exp)`. So the condition is true only if its foldable and not a literal.